contrib/scripts/avocado-run-inplace.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash -e
+#
+# shell command line helper, wrap command line ops to avocado friendly way
+#
+# Example:  avocado-run-inplace.sh dnf update -y
```

Comment:
Ok, commit looks good. Could you please send it alone, in a separate PR? Thanks!